config_defaults.yml

```diff
@@ -46,29 +46,23 @@ Gemfile:
       - gem: parallel_tests
       - gem: rubocop
         version: '0.41.2'
-        ruby-operator: '<'
-        ruby-version: '2.0.0'
+        condition: "RUBY_VERSION < '2.0.0'"
```

Comment:
I know we don't do it yet, but using `Gem::Version` would be great for correct semver comparisons.